src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Media/Animation/SequentialAnimationsPage.xaml.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System;
+using System.Threading.Tasks;
+using Uno.UI.Samples.Controls;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Media.Animation;
+
+namespace SamplesApp.Windows_UI_Xaml_Media.Animation
+{
+	[SampleControlInfo("Animations", "Sequential Animations")]
+	public sealed partial class SequentialAnimationsPage : Page
+	{
+		private readonly TimeSpan DURATION = TimeSpan.FromMilliseconds(250);
+
+		public string Status
+		{
+			get => (string)GetValue(StatusProperty);
+			set => SetValue(StatusProperty, value);
+		}
+
+		public static readonly DependencyProperty StatusProperty =
+			DependencyProperty.Register(
+				nameof(Status),
+				typeof(string),
+				typeof(SequentialAnimationsPage),
+				new PropertyMetadata(string.Empty));
+
+		public SequentialAnimationsPage()
+		{
+			this.InitializeComponent();
+
+			// Simply kick-off the task
+#pragma warning disable CS4014
+			this.Loaded += (sender, e) => StartAnimations();
+#pragma warning restore CS4014
+		}
+
+		internal void TranslateX(UIElement element, double to)
+			=> Animate(element, "(UIElement.RenderTransform).(TranslateTransform.X)", to);
+
+		internal void TranslateY(UIElement element, double to)
+			=> Animate(element, "(UIElement.RenderTransform).(TranslateTransform.Y)", to);
+
+		internal void ScaleX(UIElement element, double to)
+			=> Animate(element, "(UIElement.RenderTransform).(ScaleTransform.ScaleX)", to);
+
+		internal void ScaleY(UIElement element, double to)
+			=> Animate(element, "(UIElement.RenderTransform).(ScaleTransform.ScaleY)", to);
+
+		internal void Rotate(UIElement element, double to)
+			=> Animate(element, "(UIElement.RenderTransform).(RotateTransform.Angle)", to);
+
+		internal void Fade(UIElement element, double to)
+			=> Animate(element, "Opacity", to);
+
+		private void Animate(UIElement element, string targetProperty, double to)
+		{
+			var anim = new DoubleAnimation()
+			{
+				To = to,
+				Duration = DURATION,
+				EasingFunction = new CubicEase() { EasingMode = EasingMode.EaseInOut }
+			};
+
+			var storyboard = new Storyboard();
+
+			Storyboard.SetTarget(anim, element);
+			Storyboard.SetTargetProperty(anim, targetProperty);
+
+			storyboard.Children.Add(anim);
+			storyboard.Begin();
+		}
+
+		private async Task StartAnimations()
```

Comment:
I'm wondering if we should also take a screenshot at each step... So we can validate that the behavior stays the same. Maybe iterating through that list using a clicked button would allow for that.

Reply:
Will do this on a seperate PR. The current test will make sure it successfully completes with no crash. My next PR on this will be with your changes.